website/docs/r/data_lake.html.markdown

```diff
@@ -0,0 +1,103 @@
+  * `storage_databases.#.collections` -     Array of objects where each object represents a collection and data sources that map to a [stores](https://docs.mongodb.com/datalake/reference/format/data-lake-configuration#mongodb-datalakeconf-datalakeconf.stores) data store.
+    * `storage_databases.#.collections.#.name` - Name of the collection.
+      * `storage_databases.#.collections.#.data_sources` -     Array of objects where each object represents a stores data store to map with the collection.
+        * `storage_databases.#.collections.#.data_sources.#.store_name` -     Name of a data store to map to the `<collection>`. Must match the name of an object in the stores array.
+        * `storage_databases.#.collections.#.data_sources.#.default_format` - Default format that Data Lake assumes if it encounters a file without an extension while searching the storeName. 
+        * `storage_databases.#.collections.#.data_sources.#.path` - Controls how Atlas Data Lake searches for and parses files in the storeName before mapping them to the `<collection>`.
+  * `storage_databases.#.views` -     Array of objects where each object represents an [aggregation pipeline](https://docs.mongodb.com/manual/core/aggregation-pipeline/#id1) on a collection. To learn more about views, see [Views](https://docs.mongodb.com/manual/core/views/).
+  * `storage_databases.#.views.#.name` - Name of the view.
+  * `storage_databases.#.views.#.source` -  Name of the source collection for the view.
+  * `storage_databases.#.views.#.pipeline`- Aggregation pipeline stage(s) to apply to the source collection.
+* `storage_stores` - Each object in the array represents a data store. Data Lake uses the storage.databases configuration details to map data in each data store to queryable databases and collections. For complete documentation on this object and its nested fields, see [stores](https://docs.mongodb.com/datalake/reference/format/data-lake-configuration#std-label-datalake-stores-reference). An empty object indicates that the Data Lake has no configured data stores.
+  * `storage_stores.#.name` - Name of the data store.
+  * `storage_stores.#.provider` - Defines where the data is stored.
+  * `storage_stores.#.region` - Name of the AWS region in which the S3 bucket is hosted.
+  * `storage_stores.#.bucket` - Name of the AWS S3 bucket.
+  * `storage_stores.#.prefix` - Prefix Data Lake applies when searching for files in the S3 bucket .
+  * `storage_stores.#.delimiter` - The delimiter that separates `storage_databases.#.collections.#.data_sources.#.path` segments in the data store.
+  * `storage_stores.#.include_tags` - Determines whether or not to use S3 tags on the files in the given path as additional partition attributes.
+
+## Import
+
+Data Lake can be imported using project ID, name of data lake and aws s3 bucket, in the format `project_id`--`name`--`aws_test_s3_bucket`, e.g.
```

Comment:
Small change:
Data Lake can be imported using project ID, name of the data lake and name of the AWS s3 bucket